Modules/Core/Common/test/itkImageGTest.cxx

```diff
@@ -207,12 +206,11 @@ TEST(Image, FillBufferOfNonEqualityComparableType)
 }
 
 template <typename ImageType>
-typename ImageType::Pointer generate_image(std::array<typename ImageType::SizeValueType, 2> size)
+typename ImageType::Pointer
+generate_image(typename ImageType::SizeType size)
```

Comment:
It's more common to pass `itk::Size` by const-reference (`const typename ImageType::SizeType &`) , rather than by value, but in practice, pass by value just works fine, for an `itk::Size`, because it is trivially copyable. 😃